library/src/main/scala/scala/scalajs/js/UndefOr.scala

```diff
@@ -199,6 +200,7 @@ final class UndefOrOps[A](val self: UndefOr[A]) extends AnyVal {
   @inline final def collect[B](pf: PartialFunction[A, B]): UndefOr[B] =
     if (isEmpty) undefined
     else pf.applyOrElse(this.forceGet, (_: A) => undefined).asInstanceOf[UndefOr[B]]
+  // scalafmt: { maxColumn = 80 }
```

Comment:
I would avoid these kind of smaller, local format changes. While for that particular piece of code, it'll probably indeed look nicer, it feels like the maintenance overhead is substantial, for not a lot of gain. (js.Function files are different there IMO).